.circleci/config.base.yml

```diff
@@ -883,6 +896,16 @@ workflows:
                 - release
                 - /release_rc\/.*/
                 - /tagged-release-without-e2e-tests\/.*/
+      - verify-versions-match:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore:
+                - beta
+                - release
+                - /release_rc\/.*/
+                - /tagged-release-without-e2e-tests\/.*/
```

Comment:
couple of things here.
1. I'd remove filters. we want that validation everywhere.
2. we should run this after publish_to_local_registry, i.e. after lerna bumps versions - we want to verify that this process happens correctly.

Reply:
Ok. I will revise. Thank you.